src/Microsoft.Health.Fhir.Core.UnitTests/Features/Operations/Export/ExportOrchestratorJobTests.cs

```diff
@@ -33,106 +33,113 @@ public class ExportOrchestratorJobTests
         private IQueueClient _mockQueueClient = Substitute.For<IQueueClient>();
         private ILoggerFactory _loggerFactory = new NullLoggerFactory();
 
-        [Theory]
-        [InlineData(ExportJobType.Patient)]
-        [InlineData(ExportJobType.Group)]
-        public async Task GivenANonSystemLevelExportJob_WhenRun_ThenOneProcessingJobShouldBeCreated(ExportJobType exportJobType)
-        {
```

Comment:
I temporarily commented this out as don't fit into new paradigm as is. Not sure whether we should invest in fixing or just remove altogether.

Reply:
I would say getting at least two tests working would be good, one for a parallel job and one for a single thread job.